R/calc_AUC.R

```diff
@@ -0,0 +1,99 @@
+require(caTools)
+
+calc_AUC <-
+  function(qp,
+           test_matrix_Xi,
+           test_matrix_Xip,
```

Comment:
I actually already have this coded in a different part of the script.  Ill upload it after fixing it now.